### DISCUSSION TITLE
不应该在海外版PCL的日志里使用中文

### DISCUSSION THREAD

**Discussion by Crotina** [Meloong-Git/PCL#4895] [提问] [answered]
现在PCL的日志，呃，是用中文记录这些事件的，然后，既然要适配到海外，日志继续使用中文可能不太好，可以把日志语言换成英语？

**Comment by wyc-26** [comment]
海外用户来 GitHub 提交 Bug
得让开发者看得懂 Log 吧
估计用户平常使用也不会去翻 Log（
~以及再去翻译 Log 工作量太大了~
另：我觉得你要到多语言分支发这个 disc
又另：其实 #4145 里面提到了，不会翻译日志的

**Comment by MoYuan-CN** [comment]
多语言适配相关 -> @tangge233

**Comment by tangge233** [comment, marked as answer]
日志没啥必要英文化：
- 开发者最看得懂中文，换成英文会给开发者增添麻烦
- 换成英文需要耗费大量时间精力进行调整，不如把时间花在修 Bug 和开发新功能（虽然已经够慢了），得不偿失

所以抱歉，不考虑把日志改成英文 Orz

**Comment by Deep-Dark-Forest** [comment]
#4145
> 不进行本地化的内容
更新日志
系统日志
